Lib/javascript/v8/javascriptinit.swg

```diff
@@ -124,6 +130,10 @@ void SWIGV8_INIT (SWIGV8_OBJECT exports, SWIGV8_OBJECT /*module*/)
 }
 
 #if defined(BUILDING_NODE_EXTENSION)
+#if (NODE_MODULE_VERSION < 0x0040)
```

Comment:
```suggestion
#if (NODE_MODULE_VERSION < 64)
```
Minor point, but from what I can see the convention for defining `NODE_MODULE_VERSION` is not as a hex number but as a decimal in node_version.h and also in the table at https://nodejs.org/en/download/releases/.

Reply:
Yeah okay, change all usage in SWIG to decimal.

Reply:
Done.